DOC-417 - Write API docs (#297)
Macrometacorp/docs: docs/api-docs/index.md

---
sidebar_position: 1
title: APIs
---

> Macrometa is an API-first company and product. We create APIs calls for every task you can perform in the Macrometa platform.

Reviewer: ```suggestion
Macrometa's GDN service offering is API-first. We create API calls for every task you can perform in the Macrometa platform.
```

> 

Reviewer: ```suggestion
We currently have two ways in which you can interact with the API: the app API reference and the docs API reference.
```

> Your main tool for using REST APIs is the API reference in the GDN web browser interface. Use the built-in API reference to run various calls and view their input and output.

Reviewer: ```suggestion
The first way of using the REST API is the API reference in the GDN web browser interface (app API). Use the built-in API reference to run various calls and view their input and output.
```

> While you can run your API calls from your favorite IDE, terminal, or other interface, you can also run them directly from our API references. In this section, pages describe how to:

Reviewer: ```suggestion
While you can run your API calls from our docs API reference, the second way of using the REST API is through your favorite IDE, terminal, or other interfaces (docs API). 

In this section, pages describe how to:
```

> - [Run API commands from the Macrometa documentation API reference](run-api-commands-docs.md).

Reviewer: ```suggestion

```

> - Run API commands from the Macrometa app API reference.

Reviewer: ```suggestion
- [Run API commands from the Macrometa app API reference](run-api-commands-app.md).
- [Run API commands from the Macrometa documentation API reference](run-api-commands-docs.md).
```
